packages/cli/src/commands/deploy/index.ts

```diff
@@ -520,6 +521,8 @@ export default async (client: Client): Promise<number> => {
   }
 
   try {
+    const noAlias = argv['--no-alias'] ?? false;
```

Comment:
```suggestion
    const autoPromote = !argv['--no-alias']
```

How about this instead?